examples/test_ex.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+cat ../../header.py $1 > test.py
+chmod +x test.py
+./test.py
```

Comment:
Looking at this again... it might be best to explicitly invoke `micropython ./test.py` here, as I don't think most users will have their system Python interpreter configured to be MicroPython (which I assume is what you have here?).

Reply:
In fact, the first line in header.py is:
```
#!/opt/bin/lv_micropython -i
```
so... this is the interpreter used to execute the script. If the user has lv_micropython at a different position, only this line needs to be adapted.

Reply:
Oops... I missed `header.py`, and only looked at the example file itself, which, of course, does not have the shebang line in it.

Thanks for pointing that out - it should be fine.